modules/storages/spec/models/storage_spec.rb

```diff
@@ -77,13 +77,21 @@
   end
 
   describe '#destroy' do
-    let(:project) { create(project) }
+    let(:project) { create(:project) }
     let(:storage) { described_class.create(test_default_attributes) }
-    let(:project_storage) { Storages::ProjectStorage.create(project: project, storage: storage) }
+    let(:project_storage) { Storages::ProjectStorage.create(project: project, storage: storage, creator: test_default_creator) }
```

Comment:
I don't like the naming `test_default_creator`. Here it can safely be named `creator`. We could also rename `test_default_attributes` to `default_attributes` (please adopt the other feature spec `admin_storages_spec.rb`, too).